### DISCUSSION TITLE
Yıkılan binaların lokasyonlarını, sahiplerini ve denetim yapanlarının paylaşılabildiği platform

### DISCUSSION THREAD

**Discussion by ohk** [acikkaynak/deprem-yardim-projesi#37] [Ideas]
Bildiğiniz üzere pek çok bina yıkıldı. Bu binaları yapan firmaların, denetimleri yapan firmaların tamamının adreslerini, kaç kişiye mezar olduklarını, sahiplerinin kimler olduğunu, fotoğraf ve kanıtların yer aldığı bir platform kurulabilir. Firma ismi, kişi ismi aratarak filtreleme özelliğinin olması hoş olacaktır. Konum bazlı filtreleme de olabilir. 

Amaç bu kişilerin herhangi bir şekilde sahibi olduğu inşaatların, denetim firmalarının bir daha satılamaz durumu gelmesi. Halkın istediği zaman bu bilgilere erişerek bu kişilerin sahibi olduğu yada denetimi yaptığı binaların satın alınmasının ve kiralanmasının önüne geçilmesi.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Ancak Legal ekibi ile değerlendirmemiz sonrasında reddedildi.
